src/components/header/Header.js

```diff
@@ -45,7 +46,10 @@ const Header = () => {
         className={`w-7 h-7 mr-4 cursor-pointer ${
           isMatched && "[&_path]:stroke-darkGrey"
         }`}
-        onClick={() => navigate("/cart")}
+        onClick={() => {
+          navigate("/cart");
```

Comment:
does setting state while navigating cause any rendering issues?

Reply:
Not that I noticed. Is there a better way to do it, cause without this, if the nav menu is opened on mobile and the cart icon is clicked, it navigates to the cart page but doesn't close the nav menu